text/processing.py

```diff
@@ -256,3 +263,28 @@ def run(self):
             file_set2 = set(fin.read().split("\n"))
         with util.uopen(self.out_file, "wt") as fout:
             fout.write("\n".join(sorted(file_set1.difference(file_set2))))
+
+
+class WriteToFileJob(Job):
+    """
+    Write a given list[str] into a text file, one entry per line
+    """
+
+    def __init__(
+        self,
+        list_to_dump,
```

Comment:
I think we do not have to be specific for lists here.

I would thus just call this parameter `content`, and then we can extend it to also support  `str` or sth. else later,
and name the job `WriteToTextFileJob`.